### ISSUE TITLE
Feature request: tool tips for languages and non-terminals

### ISSUE THREAD

**Issue by florence** [racket/redex#60]
It would be nice if, when one moused over a language name or a non-terminal in a pattern, a tool tip popped up showing what was in that language/NT.

(nanopass does this and I've found it useful)

**Comment by rfindler** [comment]
You mean the productions?

Can you share a nanopass example so I can see?

Robby

**Comment by florence** [comment]
Yes those.

If you look at https://github.com/akeep/nanopass-framework-racket/blob/master/tests/coresheme-to-javascript-compiler.rkt and mouse over the LCS at line 102 you can see what its doing for languages.

**Comment by rfindler** [comment]
I watched your codewalk today and I see little value in those things. Before seeing that, I had imagined that it was on the non-terminals, not the language positions, and that seemed like too much clutter. But on the references to the language identifier, I felt like it was a lot of stuff such that you rarely saw the whole thing and it just overall didn't feel useful.

I'm going to close this, but if you want to actually implement it and try it out, that'll change my tune.
